### DISCUSSION TITLE
Perminant Dark Mode

### DISCUSSION THREAD

**Discussion by Binary-Bear** [PartialVolume/shredos.x86_64#103] [General]
Is it possible to make dark mode permanent by editing the grub.cfg perhaps?
If so would someone please explain how to do it?
Thank you.

**Comment by PartialVolume** [comment]
Enabling dark mode in the grub.cfg is not currently possible as selecting dark mode isn't a command line option in nwipe. However, It's a feature that could be added quite easily.

I would suggest you open a issue over on nwipes's GitHub page called something like Add dark mode as a command line option.

Once that's implemented then you will be able to have nwipe default to dark mode by editing grub.cfg in ShredOS.

**Comment by Binary-Bear** [comment]
Hello PartialVolume
Thank you for your reply and also for ShredOS!
ShredOS is really impressive and it's great to see some active development.
Thank you.
